x-pack/plugins/file_upload/kibana.json

```diff
@@ -4,6 +4,6 @@
   "kibanaVersion": "kibana",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["data", "usageCollection"],
+  "requiredPlugins": ["data", "security", "usageCollection"],
```

Comment:
Should this be an optional plugin instead? I'd expect we would want file upload to work even if security was disabled.